doc/source/conf.py

```diff
@@ -840,4 +840,11 @@ def setup(app):
     "pandas.zip",
 ]
 
-linkcheck_ignore = [re.escape(link) for link in links]
+# Exclude specific links from the links list
```

Comment:
What about to just do

```python
linkcheck_ignore = [
    "^http://$", 
    "^https://$",
    *[
        re.escape(link)
        for link in [
            "http://scatterci.github.io/pydata/pandas",
            "http://specs.frictionlessdata.io/json-table-schema/",
            ...
        ]
     ]
]
```

Reply:
can't we omit the `*` ?

Could you tell  why my code failed ?

Reply:
> can't we omit the `*` ?

Nope. The `*` before the `[...]` unpacks it. Meaning, we get a flat list of strings. Try printing `[1, 2, [3, 4]]` and `[1, 2, *[3, 4]]` to see the difference.

> Could you tell why my previous code failed ?

The filtering logic is correct, but ultimately what counts is the value of `linkcheck_ignore`. And you excluded `"^http://$"` and `"^https://$"` from it.